functions.php

```diff
@@ -39,18 +41,41 @@ function formatName(string $name): string
     return $name;
 }
 
-function validateInfo(string $mouseName,string $mouseBrand,string $mouseWeight): string
+function checkInputLength(string $mouseName, string $mouseBrand, int $mouseWeight): bool
+{
+    if ((strlen($mouseName) > 100) || (strlen($mouseBrand) > 50) || ($mouseWeight > 999) || ($mouseWeight < 1)) {
+        return true;
+    } else {
+        return false;
+    }
+
+}
+
+function validateWeight(int $mouseWeight): bool
+{
+    if (!(filter_var($mouseWeight, FILTER_VALIDATE_INT))) {
```

Comment:
same as above comment

Reply:
yeah i was thinking that tbh haha, i was trying to split my previous function into more but it seemed pointless but did it anyway